content/docs/guides/neon-authorize.md

```diff
@@ -198,7 +206,7 @@ Here is a non-exhaustive list of authentication providers. The table shows which
 | **Supabase Auth**      | ❌         | N/A                                                                                                                                                            | N/A                                                                                                                           |
 | **Amazon Cognito**     | ✅         | <span style={{ whiteSpace: "normal", wordBreak: "break-word" }}>`https://cognito-idp.{region}.amazonaws.com/{userPoolId}/.well-known/jwks.json`</span>         | [docs](https://docs.aws.amazon.com/cognito/latest/developerguide/amazon-cognito-user-pools-using-tokens-verifying-a-jwt.html) |
 | **Azure AD**           | ✅         | <span style={{ whiteSpace: "normal", wordBreak: "break-word" }}>`https://login.microsoftonline.com/{tenantId}/discovery/v2.0/keys`</span>                      | [docs](https://learn.microsoft.com/en-us/entra/identity-platform/access-tokens)                                               |
-| **GCP Cloud Identity** | ❌         | N/A                                                                                                                                                            | N/A                                                                                                                           |
+| **GCP Cloud Identity** | ��         | N/A                                                                                                                                                            | N/A                                                                                                                           |
 | **Descope Auth**       | ✅         | <span style={{ whiteSpace: "normal", wordBreak: "break-word" }}>`https://api.descope.com/{YOUR_DESCOPE_PROJECT_ID}/.well-known/jwks.json`</span>               | [docs](https://docs.descope.com/project-settings/jwt-templates)                                                               |
 
```

Comment:
We now support Firebase and Google Identity through the `audience`  field + JWKS combo. Maybe @davidgomes can speak his mind about how he envisioned it being laid on the docs

Reply:
Addressed by this separate PR: https://github.com/neondatabase/website/pull/2587